src/screens/LocalConfirmNotificationCodeScreen.tsx

```diff
@@ -21,27 +19,19 @@ type State = {
 };
 
 interface Props {
-  navigation: StackNavigationProp<RootStackParams, Route.ConfirmNotificationCode>;
-  createNotificationEmail: Function;
-  route: RouteProp<RootStackParams, Route.ConfirmNotificationCode>;
-  email: string;
+  navigation: StackNavigationProp<RootStackParams, Route.LocalConfirmNotificationCode>;
+  route: RouteProp<RootStackParams, Route.LocalConfirmNotificationCode>;
   pin: string;
 }
 
-class ConfirmNotificationCodeScreen extends PureComponent<Props, State> {
+class LocalConfirmNotificationCodeScreen extends PureComponent<Props, State> {
```

Comment:
Why can't we have one screen with ConfirmEmailScreen with different behavior on pressable?

Reply:
it would be hard to do that and would require much "ifology" inside component which I think would decrease readability significantly. The behaviors of that screens are very different and I think we shouldn't force them to be together